projects/centre_console/README.md

```diff
@@ -0,0 +1,38 @@
+# Centre console
+
+## What is the purpose of the board?
+The centre console is responsible for:
+* Handling the physical button interface so the driver can change from neutral, reverse, and drive
+* Powers the Raspberry Pi responsible for the driver display
+* Controls fans to cool the Raspberry Pi
+* Toggles the BPS indicator
+* Controls the hazards

```

Comment:
IMO you should also mention that centre console manages pretty much the entire state of the firmware, it's the coordinator and source of truth for all the car-level state and manages all state transitions (like turning the car on/off).